osu.Game/Screens/Menu/IntroSequence.cs

```diff
@@ -205,6 +205,7 @@ public void Start(double length)
                 const int line_end_offset = 120;
 
                 smallRing.Foreground.ResizeTo(1, line_duration, Easing.OutQuint);
+                smallRing.Delay(400).FadeOut();
```

Comment:
On closer inspection in test browser at say, 20% speed, this looks wrong as it's an instant fade. Unsure if worth fixing it but fading to black might be a better solution.

```suggestion
                smallRing.Delay(400).FadeColour(Color4.Black, 300);
```

(Can't be a nonzero duration fade either because of how `Ring` is implemented - its individual layers are getting the alpha change together instead of just the background)